### PR TITLE
QGIS vector graphics - tables for algorithm parameters

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/graphics.rst [hunk 1]
```diff
@@ -18,56 +18,129 @@ Creates a bar plot from a category and a layer field.
...
+       
+       Default: 0
+     - Additional statistics information to add to the plot.
+       One of:
+       
+       * 0 --- Show Mean
+       * 1 --- Show Standard Deviation
+       * 2 --- Don't show mean and standard deviation
+
+   * - **Bar plot**
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/graphics.rst]:
```suggestion
   * - **Box plot**
```

File: source/docs/user_manual/processing_algs/qgis/graphics.rst [hunk 2]
```diff
@@ -80,21 +153,56 @@ Creates a box plot with mean and standard deviation values.
...
+.. list-table::
+   :header-rows: 1
+   :widths: 20 20 20 40
+   :stub-columns: 0
+
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **Input layer**
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/graphics.rst]:
```suggestion
   * - **Input table**
```
even though like the others, it accepts geometric layer only. An issue in QGIS, probably!

File: source/docs/user_manual/processing_algs/qgis/graphics.rst [hunk 3]
```diff
@@ -80,21 +153,56 @@ Creates a box plot with mean and standard deviation values.
...
+     - ``INPUT``
+     - [vector: any]
+     - Input vector layer
+   * - **Category field name**
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/graphics.rst]:
```suggestion
   * - **Category name field**
```

File: source/docs/user_manual/processing_algs/qgis/graphics.rst [hunk 4]
```diff
@@ -18,56 +18,129 @@ Creates a bar plot from a category and a layer field.
...
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/graphics.rst]:
```suggestion
   * - **Category name field**
```

File: source/docs/user_manual/processing_algs/qgis/graphics.rst [hunk 5]
```diff
@@ -110,21 +218,56 @@ a numeric one).
...
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/graphics.rst]:
```suggestion
   * - **Category name field**
```

File: source/docs/user_manual/processing_algs/qgis/graphics.rst [hunk 6]
```diff
@@ -161,28 +340,64 @@ Vector layer histogram
 ----------------------
 Generates a histogram with the values of the attribute of a vector layer.
 
-The attribute to use for computing the histogram must be a numeric attribute.
+The attribute to use for computing the histogram must be numeric.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/graphics.rst]: Just citing the documentation here. And in the user interface, only numerical attributes are available for selection in "Attribute".

**Pull request merged**